subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/transform/DefaultTransformInvocationFactoryTest.groovy

```diff
@@ -272,7 +272,7 @@ class DefaultTransformerInvocationFactoryTest extends AbstractProjectBuilderSpec
     def "executes transformations in workspace (#transformationType)"(TransformationType transformationType) {
         def inputArtifact = temporaryFolder.file("input")
         inputArtifact.text = "my input"
-        def transformer = TestTransformer.create { input, outputDir ->
+        def transformer = TestTransform.create { input, outputDir ->
```

Comment:
```suggestion
        def transform = TestTransform.create { input, outputDir ->
```

:question: Could be transform, like a few other occurrences of `transformer` in this class.